.coderabbit.yaml

```
@@ -0,0 +1,40 @@
+  language_filters:
+    # TypeScriptファイルは積極的にレビュー
+    - 'typescript'
+    - 'javascript'
+    - 'tsx'
+    - 'jsx'
+
+# AI機能の設定
+ai:
+  # レビューの詳細度
+  review_depth: "standard"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)